apps/evm/src/types/index.ts

```diff
@@ -298,8 +298,9 @@ export interface Market {
 }
 
 export interface MarketSnapshot {
-  blockNumber: number;
-  blockTimestamp: number;
+  // we are migrating these values to strings, as they are bigints instead of numbers
```

Comment:
Wouldn't it make more sense for the backend to return strings directly? These values shouldn't need the range of `BigInt`.

Reply:
The range of number is big enough is most cases, that's true. But potentially you can have those running in the bigint range, in fact once we adopted viem on the backend they return these values as bigints, so I decided to adopt them instead of casting back to a number

Reply:
Ok I see. Thanks for the explanations: that works for me.